purchases/src/test/java/com/revenuecat/purchases/PurchasesCoroutinesTest.kt

```diff
@@ -2,12 +2,14 @@ package com.revenuecat.purchases
 
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import io.mockk.verify
+import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.test.runTest
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.robolectric.annotation.Config
 
+@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class, ExperimentalCoroutinesApi::class)
```

Comment:
Yeah, it's needed to remove the `runTest` warning:

> This declaration needs opt-in. Its usage should be marked with '@kotlinx.coroutines.ExperimentalCoroutinesApi' or '@OptIn(kotlinx.coroutines.ExperimentalCoroutinesApi::class)'

```kotlin
@MustBeDocumented
@Retention(value = AnnotationRetention.BINARY)
@Target(
    AnnotationTarget.CLASS,
    AnnotationTarget.ANNOTATION_CLASS,
    AnnotationTarget.PROPERTY,
    AnnotationTarget.FIELD,
    AnnotationTarget.LOCAL_VARIABLE,
    AnnotationTarget.VALUE_PARAMETER,
    AnnotationTarget.CONSTRUCTOR,
    AnnotationTarget.FUNCTION,
    AnnotationTarget.PROPERTY_GETTER,
    AnnotationTarget.PROPERTY_SETTER,
    AnnotationTarget.TYPEALIAS
)
@RequiresOptIn(level = RequiresOptIn.Level.WARNING)
public annotation class ExperimentalCoroutinesApi
```

I can remove it if you prefer so, as it's not mandatory.

Reply:
Yeah I think this makes sense. We probably won't get many users if they need to use this annotation, but if the API is subject to change, that's how it should be 👍

Reply:
Interesting, because I get that message (`This declaration needs opt-in...`) in the annotation itself not in the `runTest`, that's why I am confused

<img width="860" alt="Screenshot 2023-06-13 at 11 37 36 AM" src="https://github.com/RevenueCat/purchases-android/assets/664544/034c26f1-e811-4b0c-b7cd-955a9a9d0c39">

Reply:
That `@OptIn` warning is different 👀 

![Screen Shot 2023-06-13 at 9 47 52 AM](https://github.com/RevenueCat/purchases-android/assets/131195486/bef5ae45-c699-4d61-8875-6f6577e03c97)

BTW note that what you're seeing are the docs from `@ExperimentalCoroutinesApi` that's not the warning you get if you don't add the `@OptIn`, if you remove it you should get it (followed by the docs) 👀 

![Screen Shot 2023-06-13 at 9 47 07 AM](https://github.com/RevenueCat/purchases-android/assets/131195486/25520b1c-de8e-46c3-878a-7e782a221d79)

Reply:
Gradle... :trollface: